sdk/python/feast/feature_store.py

```diff
@@ -80,6 +80,7 @@
 from feast.infra.provider import Provider, RetrievalJob, get_provider
 from feast.infra.registry.base_registry import BaseRegistry
 from feast.infra.registry.registry import Registry
+from feast.infra.registry.snowflake import SnowflakeRegistry
```

Comment:
I'd move this import to line 172 right above `self._registry = SnowflakeRegistry(...)` so that it's not necessary for users to have installed the `snowflake` extra.